runtime/near-vm-runner/src/logic/logic.rs

```diff
@@ -1223,6 +1223,15 @@ impl<'a> VMLogic<'a> {
         self.gas_opcodes(gas)
     }
 
+    // TODO!!! Hide this behind a feature flag!
+    pub fn sleep_nanos(&mut self, nanos: u64) -> Result<()> {
```

Comment:
I wonder if we can just use the compile time feature gating for this?  Or would that not work for your usecase?

Reply:
I think it would here, but doesn't in the imports.rs because it's inside of some macro.

Reply:
https://github.com/near/nearcore/pull/9317/files#diff-e9b653ec008c4aeece3871e171113ef743d50a1d8229bddf14d1ea4c51391675R286 is an example of how to do it in `imports.rs`?

Reply:
Ah nice, I missed that, and there is another example with "sandbox" just above. I'll try this, thanks!

Reply:
I think its better to always use parameters for contract runtime nowadays. You can follow the example of yield_resume from https://github.com/near/nearcore/commit/47020d42f44ab37eeca65483a4d61b0e7b9f673d.

Reply:
Not really. The parameter files are baked into the binary at compile time. From within rust code this is pretty easy, all these parameters end up as fields in `VMConfig` that gets passed around all over. From outside I don't think there's a way unless we modify the interface of the neard binary in some way.

I don't think adding another compile time feature is the right way to go about solving this problem, but I also don't know how to solve it in a straightforward way that allows to achieve what you want in pytests.